### PR TITLE
First pass aadding cdi-full group to @Decorators tests

### PR DESCRIPTION
**Pull request by starksm64**
Addresses #263

Moves Decorators tests to org.jboss.cdi.tck.tests.full.decorators top package
Adds cdi-full test group
Adds minimal bean defining annotations for weld master to pass the tests

[INFO] Tests run: 66, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.894 s - in TestSuite
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 66, Failures: 0, Errors: 0, Skipped: 0

Signed-off-by: Scott M Stark <starksm64@gmail.com>

### CODE REVIEW

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/finalBeanClass/AfterBeanDiscoveryObserver.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.decorators.custom.broken.finalBeanClass;
+package org.jboss.cdi.tck.tests.full.decorators.broken.finalBeanClass;
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/finalBeanClass/AfterBeanDiscoveryObserver.java:17]: `decorators.custom` should IMHO be moved to `full.decorators.custom`, not `full.decorators`.

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/instance/BuiltinInstanceDecoratorTest.java [hunk 2]
```diff
@@ -15,8 +15,9 @@
  * limitations under the License.
  */
 
-package org.jboss.cdi.tck.tests.decorators.builtin.instance;
+package org.jboss.cdi.tck.tests.full.decorators.instance;
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/instance/BuiltinInstanceDecoratorTest.java:18]: `decorators.builtin` should IMHO move to `full.decorators.builtin`, not `full.decorators`.

**Review by Ladicek — APPROVED** [review]
This looks OK, baring the 2 sub-optimal package moves. But I don't feel confident about the `beans.xml` changes.

**Comment by starksm64** [comment]
The two package comments are probably refactor errors. I agree with those so I'll update those.

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/finalBeanClass/AfterBeanDiscoveryObserver.java (hunk 1 above)
Comment (starksm64) [reply impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/finalBeanClass/AfterBeanDiscoveryObserver.java]: Done

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/instance/BuiltinInstanceDecoratorTest.java (hunk 2 above)
Comment (starksm64) [reply impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/instance/BuiltinInstanceDecoratorTest.java]: Done

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/custom/finalBeanClass/AfterBeanDiscoveryObserver.java [hunk 3]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.decorators.custom.broken.finalBeanClass;
+package org.jboss.cdi.tck.tests.full.decorators.broken.custom.finalBeanClass;
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/custom/finalBeanClass/AfterBeanDiscoveryObserver.java:17]: Should be `full.decorators.custom.broken`, not `full.decorators.broken.custom`.

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/nodelegateinjectionpoint/AfterBeanDiscoveryObserver.java [hunk 4]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.decorators.custom.broken.nodelegateinjectionpoint;
+package org.jboss.cdi.tck.tests.full.decorators.broken.nodelegateinjectionpoint;
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/nodelegateinjectionpoint/AfterBeanDiscoveryObserver.java:17]: Should be `full.decorators.custom.broken.nodelegateinjectionpoint`, not `full.decorators.broken.nodelegateinjectionpoint`.

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/toomanydelegateinjectionpoints/AfterBeanDiscoveryObserver.java [hunk 5]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.decorators.custom.broken.toomanydelegateinjectionpoints;
+package org.jboss.cdi.tck.tests.full.decorators.broken.toomanydelegateinjectionpoints;
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/toomanydelegateinjectionpoints/AfterBeanDiscoveryObserver.java:17]: Should be `full.decorators.custom.broken.toomanydelegateinjectionpoints`, not `full.decorators.broken.toomanydelegateinjectionpoints`.

**Comment by Ladicek** [comment]
Just spent some time looking at the `beans.xml` changes and they look fine, Git just (once again) didn't understand what's going on.

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/custom/finalBeanClass/AfterBeanDiscoveryObserver.java (hunk 3 above)
Comment (starksm64) [reply impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/custom/finalBeanClass/AfterBeanDiscoveryObserver.java:17]: Changed.

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/toomanydelegateinjectionpoints/AfterBeanDiscoveryObserver.java (hunk 5 above)
Comment (starksm64) [reply impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/toomanydelegateinjectionpoints/AfterBeanDiscoveryObserver.java:17]: Changed.

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/nodelegateinjectionpoint/AfterBeanDiscoveryObserver.java (hunk 4 above)
Comment (Ladicek) [reply impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/nodelegateinjectionpoint/AfterBeanDiscoveryObserver.java:17]: The other issues are fixed, but this one remains.

**Review by manovotn — APPROVED** [review]
Looks pretty good

File: impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/nodelegateinjectionpoint/AfterBeanDiscoveryObserver.java (hunk 4 above)
Comment (Ladicek) [reply impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/nodelegateinjectionpoint/AfterBeanDiscoveryObserver.java:17]: This wasn't addressed if I understand correctly.

Comment (starksm64) [reply impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/nodelegateinjectionpoint/AfterBeanDiscoveryObserver.java:17]: Ok, #280 add this additional rename

**Pull request merged**